docs/deploy/howto-add-aks-to-ps1.md

```diff
@@ -309,7 +309,7 @@ Note the following values in the YAML file:
 
 ```yaml
 image:
-  tag: 2.8.3
```

Comment:
We usually update those values only after we have a corresponding release. Please see #1748 for a sample of such change.

Reply:
Will do